hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -1100,6 +1101,42 @@ final Stream<DynamicTest> duplicateAirdropsFail() {
                         .hasPrecheck(PENDING_AIRDROP_ID_REPEATED)));
     }
 
+    @HapiTest
+    @DisplayName("account created with same alias should fail")
+    final Stream<DynamicTest> accountCreatedWithSAmeAliasShouldFail() {
+        final String ALIAS = "alias";
+        return hapiTest(
+                // add common entities, you can create your own ones
+                flattened(
+                        setUpTokensAndAllReceivers(),
+                        // stop the unlimitedAutoAssociations, in order to create the account and send the airdrop to
+                        // pending state
+                        // you can create the first aliased account in setUpEntitiesPreHIP904(), instead of overriding
+                        // here
+                        overriding("entities.unlimitedAutoAssociationsEnabled", "false"),
+
+                        // create key
+                        newKeyNamed(ALIAS),
+                        // create first aliased account
+                        cryptoTransfer(tinyBarsFromAccountToAlias(OWNER, ALIAS, 1)),
+                        // save aliased account into registry
+                        withOpContext((spec, opLog) -> updateSpecFor(spec, ALIAS)),
+                        // airdrop
+                        tokenAirdrop(moving(1, FUNGIBLE_TOKEN).between(OWNER, ALIAS))
+                                .payingWith(OWNER),
+                        // airdrop should be pending, you can assert txn record too
+                        getAccountBalance(ALIAS).hasTokenBalance(FUNGIBLE_TOKEN, 0),
+                        // delete the account
+                        cryptoDelete(ALIAS),
+                        // create new account with the same key
+                        cryptoTransfer(tinyBarsFromAccountToAlias(OWNER, ALIAS, 1)),
+                        withOpContext((spec, opLog) -> updateSpecFor(spec, ALIAS)),
+                        // try to claim
+                        tokenClaimAirdrop(pendingAirdrop(OWNER, ALIAS, FUNGIBLE_TOKEN))
+                                .signedBy(ALIAS, DEFAULT_PAYER)
+                                .hasKnownStatus(INVALID_PENDING_AIRDROP_ID)));
```

Comment:
CLAIM_69 says should fail with INVALID_SIGNATURE. Then CLAIM_36 is the same scenario, expect success. Which one is correct? @Neeharika-Sompalli @anastasiya-kovaliova

Reply:
yes this makes sense, because the AccountID of receiver will be different.

Reply:
Then we should update the test plan on CLAIM_69 and CLAIM_36. @anastasiya-kovaliova